docs/source/setup/gee.rst

```diff
@@ -1,174 +1,178 @@
 
 .. image:: ../_images/setup/gee/gee_disconnected.png
-    :alt: connection popup
+    :alt: connection pop-up
 
-Click on :code:`Connect your Google account`. Google is now redirecting you to its connection page. in the given list of google account connect yourself to the one you use with GEE: 
+Click on :code:`Connect your Google account`. In the list provided, select the account associated with GEE: 
 
 .. image:: ../_images/setup/gee/gee_credential.png
 
-.. warning::
+.. Note::
 
-    You can authorized everything Google will be asking you. It will authorized SEPAL to interact with your Google Drive and Asset home folder to export recipes and modules results.
+    You will see a message requesting your permission to authorize SEPAL's access to your Google Drive, as well as your **Asset** and **Home** folders to export recipes and module results.
 
-After this connection process, you will be redirected to SEPAL main page. If you click again on the user interface button, the popup window will display "Connected" in green at the top right side.
+After the connection process is complete, you will be redirected to the SEPAL website. If you click on the "User interface" button again, the pop-up window will display "Connected" in green on the upper-right side.
 
 .. image:: ../_images/setup/gee/user_interface_connected.png
-    :alt: sepal and gee connected
+    :alt: SEPAL and GEE connected
 
 Disconnection
 ^^^^^^^^^^^^^
 
-This operation is of course reversible so if you want to change account or simply remove the link between your SEPAL account and your GEE one you can disconnect SEPAL from the GEE service. 
+If you want to change accounts or remove the link between your SEPAL and GEE accounts, you can disconnect SEPAL from GEE at any time. 
 
 Return to the user interface and click on :code:`Google account`. You should see the following window: 
 
 .. image:: ../_images/setup/gee/gee_connected.png
     :alt: gee connected 
 
-Simply click on the :code:`disconnect your Google account` Button and you're done! 
+Simply click on the :code:`Disconnect your Google account` button and your accounts will no longer be linked. 
 
 Upload files to GEE 
 -------------------
 
-When you'll be using our recipes and modules, you'll be asked to use Google EE assets to provide rasters or vectors to the core GEE based algorithm. These assets can be public assets that someone shared with you but it can also be assets that you created yourself and you would like to use.
+When you use SEPAL recipes and modules, you'll be asked to use GEE assets to provide rasters or vectors to the core GEE-based algorithm. These assets can be either public assets that someone shared with you or assets that you created yourself and would like to use.
```

Comment:
```suggestion
When you use SEPAL recipes and modules, you'll be asked to use GEE assets to provide rasters (:code:`ee.Image`) or vectors (:code:`ee.FeatureCollection`) to the core GEE-based algorithm. These assets can be either public assets that someone shared with you or assets that you created yourself and would like to use.
```

putting the names that people already know (raster and vector) in correspondance with what they will use (Image and FeatureCollection). what do you think ?